product_docs/docs/pem/8/tuning_performance/postgres_expert.mdx

```diff
@@ -9,64 +9,62 @@ redirects:
+Select or unselect the servers and databases that you want to evaluate. If you select multiple servers or databases, then the resulting report contains a separate evaluation of each target. When you've finished, select **Next** to continue.
 
 ![Specify a report destination](../images/pe_direct_output.png)
 
-You can select the default option and click `Finish` to view an onscreen report from Postgres Expert, or check the box next to Download the report to save a copy of the report to an HTML file for later use. If you choose to save the report to a file, the download will begin immediately. The file will be saved in your default download directory.
+To view the report in the client, select **View the report now**. To save a copy to an HTML file, select **Download the report**. If you choose to download the report, then the file saves to your default download directory. Select **Finish**. 
 
 ## Reviewing Postgres Expert recommendations
 
-Postgres Expert produces an easily navigated report that contains an analysis of the selected rules, categorized by high, medium, and low severities, for the selected servers.
+Postgres Expert produces an easy-to-navigate report that contains an evaluation of the selected rules. Each rule is categorized as high, medium, or low severity for the selected servers.
 
 ![The Postgres Expert report](../images/pe_report.png)
 
-The report header contains a summary of the report, and includes the date and time that the report was generated, the number of rules analyzed, and the number of deviations from best practices found by Postgres Expert. Use the `Jump to` drop-down listbox to select a server to navigate to the section of the report that targets recommendations for that server.
+The report header contains a summary of the report that includes:
+
+- The date and time of the report
+- The number of rules evaluated
+- The number of deviations from best practices
+
+To jump to the section for a given server, select the server from the **Go to** list.
 
-The body of the report contains the detailed findings for each server selected for analysis. The findings are sorted by Expert; within each Expert heading, any rule violations are ranked by Severity.
+The body of the report contains the detailed findings for each of the selected servers. The findings are sorted by expert. Within each expert heading, any rule violations are ranked by severity.
 
 ![The detailed recommendation for a rule](../images/pe_param_value.png)
 
-Click on each rule in the Postgres Expert report to display details and recommendations for that rule. Within each rule, section headings display:
+Select each rule in the report to display details and recommendations for that rule. Within each rule, there are unique sections:
```

Comment:
"Within each rule" puzzled me. I think it would be clearer as:
Selecting the rule displays these section headings: